### PR TITLE
fix: use https in schemas

### PR DESCRIPTION
**Pull request by spaenleh**
To fix Sonar cloud quality issues I update all the schema to https.

### CODE REVIEW

**Review by ReidyT — APPROVED** [review]
Nice 🏢

File: .devcontainer/docker-compose.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ services:
       # the Localstack config is set by the "localstack" service below
       S3_FILE_ITEM_HOST: http://localstack:4566
       # the Iframely config is set by the "iframely" service below
-      EMBEDDED_LINK_ITEM_IFRAMELY_HREF_ORIGIN: http://localhost:8061
+      EMBEDDED_LINK_ITEM_IFRAMELY_HREF_ORIGIN: https://localhost:8061
```
Comment (pyphilia) [inline .devcontainer/docker-compose.yml:45]: You do it for localhost but not localstack? Does it still work?

**Review by pyphilia — APPROVED** [review]
Maybe take a serious look at the comment, otherwise LGTM thank you!

**Comment by spaenleh** [comment]
@pyphilia good catch ! I did not intend to change the localhost protocol. Fixed.

**Pull request merged**